.github/workflows/main.yml

```diff
@@ -0,0 +1,67 @@
+name: CI
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+# events but only for the master branch
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install Bazelisk
+        run: |
+          wget https://github.com/bazelbuild/bazelisk/releases/download/v1.6.1/bazelisk-linux-amd64
+          echo "76c3af8cfe19f287b532990296bb58f02074d388856256b45a5f51bdbd196657 bazelisk-linux-amd64" | sha256sum --check
+          mkdir -p "${GITHUB_WORKSPACE}/bin/"
+          sudo mv bazelisk-linux-amd64 ${GITHUB_WORKSPACE}/bin/bazelisk
+          chmod +x ${GITHUB_WORKSPACE}/bin/bazelisk
+      - name: Build sources
+        run: |
+          "${GITHUB_WORKSPACE}/bin/bazelisk" build //src/...
+  # This workflow contains a single job called "build"
+  formatting:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install Bazelisk
```

Comment:
Why do you have to repeat install Bazelisk two times after the first time? If you really need to repeat it, I suggest to use YaML references so that you won't make copy-paste inconsistencies later.

```yaml
myvar: foovalue &MYREF
othervar: *MYREF
```
Both will have `foovalue` in ^.

Reply:
This allows formatting, testing etc to run in parallel.

Reply:
https://github.community/t/support-for-yaml-anchors/16128 cannot fix it with anchors yet